sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1032,7 +1032,12 @@ abstract class SQLQuerySuiteBase extends QueryTest with SQLTestUtils with TestHi
       val data = (1 to 100000).map { i => (i, i, i) }
       data.toDF("d1", "d2", "d3").createOrReplaceTempView("script_trans")
       assert(0 ===
-        sql("SELECT TRANSFORM (d1, d2, d3) USING 'cat 1>&2' AS (a,b,c) FROM script_trans").count())
```

Comment:
We cannot accept this query after this PR?

Reply:
> We cannot accept this query after this PR?

Since behavior changed after current PR. I will revert this change after follow @HyukjinKwon 's  idea.